conf/cmi/language/fi/field.field.paragraph.banner.field_banner_link.yml

```diff
@@ -1,2 +1 @@
 label: Linkki
-description: 'You can add link button to the banner using this field.'
```

Comment:
Most of the field description translations are being deleted. Was the translations imported correctly during installation? Try to run `drush locale:check && drush locale:update`.

Reply:
This worked so I was able to add the Finnish texts there now <3